strategies/indicators/HEIKEN.js

```diff
@@ -0,0 +1,61 @@
+/*
+
+  Heiken-Ashi - manuel.wassermann97@gmail.com 21/01/2018
+
+ */
+
+
+var Indicator = function(){
+  this.input = 'candle';
+  this.high = null;
+  this.low = null;
+  this.open = null;
+  this.close = null;
+}
+
+
+
+Indicator.prototype.update = function(candle) {
+
+  // Only calculate if we have an previous candle
+  if(this.lastCandle){
+    
+    // heiken.close
+    this.xClose = function(previous, current){
+      return (previous.close+current.open+current.low+current.high)/4;
+    }  
+    
+    // first heiken.open 
+    // This is probably wrong
+    first = true;
+    if(first){
```

Comment:
Seems like first will always be true. ?

Reply:
We ever hear anythings back on this ? It will always be true. Anyone else got a working Heiken Ashi Indicator they can point me to ?